src/Components/HomePage/HomePage.js

```diff
@@ -2,39 +2,27 @@ import React, { useState } from "react";
 import ImageUpload from "../ImageUpload/ImageUpload";
 import ErrorPage from "../ErrorPage/ErrorPage";
 import "./HomePage.css";
-import { mockMushroomCards } from '../../mockData.js';
-import { Link } from 'react-router-dom';
-import MushroomCard from "../MushroomCard/MushroomCard";
 
 
-const HomePage = ({ error, userImage, onImageUpload }) => {
-  // const [mushrooms, setMushrooms] = useState([]);
-
-  const renderMushroomCards = () => {
-    return mockMushroomCards.map(mushroom => (
-      <Link to={`details/${mushroom.id}`} key={mushroom.id}> 
-        <MushroomCard
-            image={mushroom.image}
-            latinName={mushroom.latinName}
-            commonNames={mushroom.commonNames}
-            probability={mushroom.probability}
-        />
-      </Link>
-    ));
-  };
+const HomePage = ({ error, userImage, onImageUpload, renderMushroomCards }) => {
 
   return (
     <div className="home-page-container">
       {userImage ? (
-        <div className="homepage-left-container">
-          <img className="user-image" src={userImage} alt="mushroom" />
-          <button
-            className="upload-new-button"
-            onClick={() => onImageUpload(null)}
-          >
-            Upload New Photo
-          </button>
-        </div>
+        <>
```

Comment:
the left container holds the user image and upload button, and the right container displays mushroom cards?